jabgui/src/main/java/org/jabref/gui/maintable/SearchShortScienceAction.java

```diff
@@ -23,6 +23,7 @@ public class SearchShortScienceAction extends SimpleCommand {
     private final StateManager stateManager;
     private final GuiPreferences preferences;
 
+
```

Comment:
Please do not add additoinal empty lines.

Maybe, the changes of this class can be reverted as only empty lines were added?